### ISSUE TITLE
Two Errors

### ISSUE THREAD

**Issue by garretthaima** [wmellema/Py-SteamCMD-Wrapper#1]
First of all this is development is great and I was banging my head against the wall using subprocess to download Arma. It worked but this is WAY cleaner.

I followed the instructions with some path modifications and I get two errors. One I think is pretty easy because I saw you weren't importing tarfile (needed for linux). The other is me getting this error: "NameError: name 'SteamCMDException' is not defined"
This is what the portion that is failing looks like:
```
s = SteamCMD(ARMA_ROOT)
try:
    s.install()
except SteamCMDException:
    print("Already installed, try to use the --force option to force installation")
```

It could be because it was never extracted because of the missing tarfile import.

**Comment by wmellema** [comment]
Good to hear that it is useful to you! I've checked the example in the readme, and I forgot to add an import. If you want to be able to catch the `SteamCMDException`, you should import it using `from pysteamcmdwrapper import SteamCMDException` at the top of your code

**Comment by wmellema** [comment]
I'll also commit a patch for the whole tarfile mistake. Somehow, it slipped through the cracks. As for the import statement, you can also replace `SteamCMD`  with a wildcard, so you import everything from the library

**Comment by wmellema** [comment]
I've released a new version (v0.0.4) that should address the above issues. Please update the version you're using with the one in version v0.0.4

**Comment by wmellema** [comment]
If the issue is resolved, please close this issue or comment so that i can close it
